chsdi/models/vector/vbs.py

```diff
@@ -160,14 +160,24 @@ class BundestankstellenBebeco:
     __table_args__ = ({'schema': 'militaer', 'autoload': False, 'extend_existing': True})
     __template__ = 'templates/htmlpopup/bundestankstellen.mako'
     __bodId__ = 'ch.vbs.bundestankstellen-bebeco'
-    __queryable_attributes__ = ['ort', 'plz', 'strasse']
+    __queryable_attributes__ = ['ort', 'standort', 'adresse', 'plz', 'produkt_de', 'produkt_fr', 'produkt_it']
```

Comment:
the list of queryable_attributes is linked to the sphinxsearch configuration [1]
please use this new list of attributes to the sphinxsearch configuration:
https://github.com/geoadmin/service-sphinxsearch/blob/ac91448e203f4f5b916c5d4abc9263f088b109f1/conf/vbs.conf.part#L82

[1] https://ltwiki.adr.admin.ch:8443/display/BGDI/03_VIEWER

Reply:
But produkt_de, produkt_fr, produkt_it are searchable. There are also in the pr in the sphinxsearch configuration... https://github.com/geoadmin/service-sphinxsearch/pull/433

Reply:
oh, i forgot that there's already a sphinxsearch pr. its all good then.